docs/src/components/Section.js

```diff
@@ -0,0 +1,47 @@
+// @flow strict
+import React, { useEffect, useState } from 'react';
+import { Box, TapArea, Text } from 'gestalt';
+import { useLocation } from 'react-router-dom';
+import Collapsable from './Collapsable.js';
+
+export default function Section({
+  section,
+  groups,
+}: {|
+  section: string,
+  groups: { [string]: Array<string> },
+|}) {
+  const { pathname } = useLocation();
+  const [collapsed, setCollapsed] = useState(
+    section === pathname.split('/')[1]
+  );
+
+  useEffect(() => {
+    setCollapsed(section === pathname.split('/')[1]);
```

Comment:
@AlbertCarreras could you add a comment to what this does?

Reply:
DONE